amundsen_application/static/.betterer.results

```diff
@@ -0,0 +1,358 @@
+    ],
+    "js/ducks/tags/reducer.ts:419330645": [
+      [130, 16, 48, "Object is possibly \'undefined\'.", "1931932323"]
+    ],
+    "js/ducks/user/reducer.ts:1917256202": [
+      [119, 22, 41, "Object is possibly \'undefined\'.", "291980569"],
+      [135, 16, 33, "Object is possibly \'undefined\'.", "625812796"],
+      [154, 15, 36, "Object is possibly \'undefined\'.", "3626509514"],
+      [171, 16, 37, "Object is possibly \'undefined\'.", "1613771502"]
+    ],
+    "js/ducks/user/tests/index.spec.ts:2988985083": [
+      [65, 13, 7, "Object is possibly \'undefined\'.", "2531329355"],
+      [84, 13, 7, "Object is possibly \'undefined\'.", "2531329355"],
+      [103, 13, 7, "Object is possibly \'undefined\'.", "2531329355"],
+      [122, 13, 7, "Object is possibly \'undefined\'.", "2531329355"]
+    ],
+    "js/ducks/utilMethods.ts:2135866627": [
+      [51, 4, 5, "Type \'string | null\' is not assignable to type \'string | undefined\'.\\n  Type \'null\' is not assignable to type \'string | undefined\'.", "173467459"]
+    ],
+    "js/fixtures/globalState.ts:3219246446": [
+      [67, 4, 12, "Type \'null\' is not assignable to type \'string\'.", "124336133"],
+      [154, 4, 10, "Type \'null\' is not assignable to type \'number\'.", "3382497788"]
+    ],
+    "js/fixtures/metadata/table.ts:2912527294": [
+      [52, 6, 11, "Type \'null\' is not assignable to type \'string\'.", "1848305091"],
+      [171, 4, 11, "Type \'null\' is not assignable to type \'string\'.", "1848305091"],
+      [183, 4, 11, "Type \'null\' is not assignable to type \'string\'.", "1848305091"],
+      [195, 4, 11, "Type \'null\' is not assignable to type \'string\'.", "1848305091"]
+    ],
+    "js/fixtures/mockRouter.ts:3563515077": [
+      [30, 6, 4, "Type \'null\' is not assignable to type \'{ (path: string, state?: {} | null | undefined): void; (location: LocationDescriptorObject<{} | null | undefined>): void; }\'.", "2088074939"],
+      [31, 6, 7, "Type \'null\' is not assignable to type \'{ (path: string, state?: {} | null | undefined): void; (location: LocationDescriptorObject<{} | null | undefined>): void; }\'.", "1364993353"],
+      [32, 6, 2, "Type \'null\' is not assignable to type \'(n: number) => void\'.", "5860973"],
+      [33, 6, 6, "Type \'null\' is not assignable to type \'() => void\'.", "1396285030"],
+      [34, 6, 9, "Type \'null\' is not assignable to type \'() => void\'.", "3244819094"],
+      [35, 6, 5, "Type \'null\' is not assignable to type \'(prompt?: string | boolean | TransitionPromptHook<{} | null | undefined> | undefined) => UnregisterCallback\'.", "175793644"],
+      [36, 6, 10, "Type \'null\' is not assignable to type \'(location: LocationDescriptorObject<{} | null | undefined>) => string\'.", "1487278680"],
+      [37, 6, 6, "Type \'null\' is not assignable to type \'(listener: LocationListener<{} | null | undefined>) => UnregisterCallback\'.", "1420496588"]
+    ],
+    "js/utils/navigationUtils.ts:1127210474": [
```

Comment:
is this file auto-gen with tool?

Reply:
Yep, it is a list of the errors we would get if the "strictNullChecks" flag was on.
It will break the build if we got worse than this file from now on!

Reply:
Do we need to commit this file for the tool to work? Or can we .gitignore this?

Reply:
Yes, we need to commit it for the script to work.
In theory, it will only get reduced!